### PR TITLE
NEXT-15838 - Update documents for request header context lang and locale

### PR DESCRIPTION
**Pull request by nam4am**
update document for shopware context language and locale user

### CODE REVIEW

File: guides/plugins/apps/app-base-guide.md [hunk 1]
```diff
@@ -104,6 +104,7 @@ GET https://my.example.com/registration?shop-id=KIPf0Fz6BUkN&shop-url=http%3A%2F
 
 {% hint style="info" %}
 Starting from Shopware version 6.4.1.0, the current shopware version will be sent as a `sw-version` header.
+Starting from Shopware version 6.4.5.0, the current shopware context language-id `sw-context-language`, and `sw-user-language` represent for the locale of the user or locale of the context language, will be sent to every request header.
```
Comment (keulinho) [inline guides/plugins/apps/app-base-guide.md:107]:
```suggestion
Starting from Shopware version 6.4.5.0, the current language id of the shopware context will be sent as a  `sw-context-language` header , and the locale of the user or locale of the context language is available under the `sw-user-language`-header.
```

please adjust the other occurences accordingly

**Comment by JoshuaBehrens** [comment]
It is cool that we have this feature now. Can we link at some other place where one can see how it is used? It is not obvious what to do with that new values without knowing a context can have a language chain.

**Comment by keulinho** [comment]
@JoshuaBehrens i've merged this PR as is right now, feel free open another PR and add some more information to make things clearer

**Comment by JoshuaBehrens** [comment]
I put it on my bucket list. Thanks for the ping.

**Pull request merged**